index.bs

```diff
@@ -59,6 +59,8 @@ urlPrefix: https://url.spec.whatwg.org/; spec: URL
     url: #percent-encode; type: abstract-op; text: percent-encode
 urlPrefix: https://www.w3.org/TR/payment-request/; spec: payment-request
     url: #dfn-state; type: dfn; text: [[state]];
+urlPrefix: https://w3c.github.io/gamepad/; spec: Gamepad
+    url: #event-gamepadconnected; type: event; text: gamepadconnected
```

Comment:
I believe this can go... bikeshed should be able to auto link it. 

```suggestion
```

Reply:
This is the same problem as before where doing so will link to the TR version :(. But last time @hober overrode our work to link to the ED so it seems unlikely we're going to get a good outcome here.